doc/dftb+/manual/dftbp.tex

```diff
@@ -3756,6 +3757,42 @@ \subsection{Casida}
     decrease memory use substantially, if you are short on memory.
     
   \end{description}
+  
+  
+\subsection{Pp-RPA}
+\label{sec:dftbp.pprpa}
+
+This tag contains the specifications for the calculation of excitation energies using the particle-particle random phase approximation (pp-RPA)~\cite{Yang2017}. This approach, unlike the time-dependent DFTB formalism, allows the computation of double and charge-transfer transitions. However, it has the limitation, that the computed excitations has to involve, at least partially, the highest occupied molecular orbital (HOMO) of the system. 
+
+For the computation of the excitation energies of a neutral $N$-electron system, one needs to set a DFTB ground state calculation for the two-electron deficient ($N-2$) system ( \kw{Charge} = 2.0 ). Please note that if \kw{Charge} is set to 0.0 (default value), the obtained transition energies will correspond to the negatively charged ($N+2$)-electron system. 
```

Comment:
Any reason not to change the default charge behaviour in the parser/initialisation such that a requested charge of 0 gives internally the correct +2 reference system?

Reply:
I think this could lead to confusions as the ground state output files would correspond to a +2 charged system while the charge specified in the input would be 0. What do you think?

Reply:
Otherwise the code would have to perform two calculations: one for the zero-charged ground state output and then a second one for the 2+ charged system needed for ppRPA.

Reply:
Does it ever make sense to obtain spectra for anything other than the neutral closed shell systems (by, as you say, simulating the +2 system)? If not, we can remove the the charge choice  option for ppRPA (and explain, with reference to your paper about the reference system calculation).

Reply:
it might make sense to calculate for charged closed-shell molecules, but I agree that the most requested calculations will probably be for neutral systems. I think we should at least provide this option: let the user set a nonzero charge, while the actual charge used for the calculations be the indicated in the input plus 2. 
what do you think? 
How do we proceed with the changes?